sdk/core/azure_core/src/lib.rs

```diff
@@ -27,7 +27,10 @@ pub use typespec_client_core::{
     fmt, json, sleep, stream, time, Bytes, Uuid,
 };
 
+/// Abstractions for distributed tracing and telemetry.
 pub mod tracing {
+    pub use crate::http::policies::PublicApiInstrumentationInformation;
+    pub use azure_core_macros::*;
```

Comment:
Might be good to be explicit here because we may add more non-tracing stuff in the future. Or maybe everything therein should be in a `tracing` submodule now (well, before GA) such that you import it like everything below on line 34.